src/app.tsx

```diff
@@ -11,7 +11,7 @@ import {
 import { ApolloProvider } from '@apollo/client';
 import client from './client';
 
-import Media from './components/media';
+import Media from './routes/Home/index';
```

Comment:
Why was this changed?

Reply:
Because I moved around files in one of my commits. Restructured things into something that makes more sense.